Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -297,3 +314,21 @@ extension PaymentViewController: SecurityCodeViewDelegate {
     delegate?.securityCodeIsUpdated(value: securityCode)
   }
 }
+
+extension PaymentViewController: UIScrollViewDelegate {
+  func scrollViewDidScroll(_ scrollView: UIScrollView) {
+    var contentOffsetY = scrollView.contentOffset.y
+
+    if #available(iOS 11.0, *) {
+      contentOffsetY += scrollView.adjustedContentInset.top
+    }
+
+    if headerView.frame.maxY > 0, contentOffsetY > headerView.frame.maxY / 2 {
```

Comment:
Can we somehow unit test this logic ? what happens when scrollViewDidScroll

Reply:
It can be tested using UI test